services/movie.py

```diff
@@ -38,3 +21,24 @@ def create_movie(
         movie.actors.set(actors_ids)
 
     return movie
+
+
+def get_movies(
+    genres_ids: Optional[List[int]] = None,
+    actors_ids: Optional[List[int]] = None,
```

Comment:
```suggestion
    genres_ids: List[int] | None = None,
    actors_ids: List[int] | None= None,
```

The code style should be the same in all code.